go.mod

```diff
@@ -145,3 +145,5 @@ require (
 	k8s.io/utils v0.0.0-20210111153108-fddb29f9d009 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.1.2 // indirect
 )
+
+replace github.com/flyteorg/flyteidl => github.com/flyteorg/flyteidl v1.3.8-0.20230221191429-648cf69c3ec8
```

Comment:
Will use release version once this is checkedin https://github.com/flyteorg/flyteidl/pull/369